.pre-commit-config.yaml

```diff
@@ -30,7 +30,7 @@ repos:
     args:
     - "--check"
     - "--diff"
-- repo: https://gitlab.com/pycqa/flake8
+- repo: https://github.com/pycqa/flake8
```

Comment:
Erm.  What's this about @ChenyuLInx ?

Reply:
This is for porting the change @MichelleArk  did for fixing the code quality check